bbaime98/free_Mentors
# #168111970 adding responsiveness

#### What does this PR do?
- add responsiveness to web pages
#### Description of Task to be completed?
- create responsiveness javascript 
- add javascript in HTML 
#### How should this be manually tested?
- clone repository 
- open any web page 
- minimize the screen and see how the page is responsive 
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#168111970
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
